snowflake/datadog_checks/snowflake/check.py

```diff
@@ -82,13 +86,8 @@ def execute_query_raw(self, query):
             return cursor.fetchall()
 
     def connect(self):
-        # verify connection is still active
-        if self._conn is not None:
-            if self._conn.is_closed():
-                self.log.warning("Connection failed, establishing a new connection.")
-            else:
-                self.service_check(self.SERVICE_CHECK_CONNECT, self.OK, tags=self._tags)
-                return
+        self.log.debug("Establishing a new connection to Snowflake.")
```

Comment:
I would suggest logging all non-sensitive informations too:

```
        self.log.debug("Establishing a new connection to Snowflake. user=%s, account=%s, account=%s, schema=%s, ....")
```

That will be very helpful for future troubleshooting.